nfc/libs/NfcCoreLib/lib/Nci/phNciNfc.c

```diff
@@ -208,18 +340,30 @@ phNciNfc_Initialise(
             wStatus =  phNciNfc_LogConnInit(pNciContext);
             if(NFCSTATUS_SUCCESS == wStatus)
             {
-                pNciContext->ResetInfo.ResetTypeReq = eResetType;
-                /* Init payload to display the build number in the init response */
-                pNciContext->tInitInfo.bExtension = 0x00;
-                pNciContext->tInitInfo.bSkipRegisterAllNtfs = 0x00;
-                wStatus = phNciNfc_GenericSequence((void *)pNciContext, NULL, NFCSTATUS_SUCCESS);
-                if(NFCSTATUS_PENDING != wStatus)
+                wStatus = phNciNfc_RegAllNtfs(pNciContext);
```

Comment:
The function call changed location in: https://github.com/Microsoft/NFC-Class-Extension-Driver/commit/ff56ba37016a1ec5610d0dc8ddfe014b628dadb5